packages/checkout-storefront/src/sections/PaymentSection/AdyenDropIn/errorMessages.ts

```diff
@@ -0,0 +1,215 @@
+import { defineMessages } from "react-intl";
+
+export const adyenErrorMessages = defineMessages({
+  refused: {
+    defaultMessage: "The transaction was refused.",
+    id: "AdyenDropIn/errorMessages/VFX2HQ",
+    description: "The transaction was refused.",
+  },
+  acquirerError: {
+    defaultMessage:
+      "The transaction did not go through due to an error that occurred on the acquirer's end.",
+    id: "AdyenDropIn/errorMessages/71wcNF",
+    description:
+      "The transaction did not go through due to an error that occurred on the acquirer's end.",
+  },
+  blockedCard: {
+    defaultMessage: "The card used for the transaction is blocked, therefore unusable.",
```

Comment:
```suggestion
    defaultMessage: "The card used for the transaction is blocked, please contact your bank to resolve this issue.",
```
If the card is blocked, it's usually due to disabling this on the bank side or having exceeded the daily transaction limit

Reply:
These messages are temporary replacements, literally copied from the adyen site. We'll be adding Apple Pay next, I'll make sure to get proper messages from Timur in that pr as well

Reply:
Can we add an issue for that, so it's not forgotten? 😅